src/core/lib/transport/transport.cc

```diff
@@ -184,7 +184,12 @@ void grpc_transport_set_pops(grpc_transport* transport, grpc_stream* stream,
              nullptr) {
     transport->vtable->set_pollset_set(transport, stream, pollset_set);
   } else {
+#ifdef GRPC_CFSTREAM
```

Comment:
Same remark as before - I don't like to introduce #ifdefs in the middle of code like this.